public/index.html

```diff
@@ -12,5 +12,20 @@
     <div id="app"></div>
     <script src="build/vendor.js"></script>
     <script src="build/app.js"></script>
+    <script>
+    if ('serviceWorker' in navigator) {
+      navigator.serviceWorker.register('./service-worker.js', {
+          scope: './'
+        })
+        .catch(function(error) {
+          console.error('Error registering service worker:' + error);
+        });
+      navigator.serviceWorker.addEventListener('controllerchange', function() {
```

Comment:
Anyway, this is not important at a moment since this app doesn't update often (e.g. every day).